src/entity.ts

```diff
@@ -554,6 +662,13 @@ export namespace entity {
    *
    * @private
    * @param {object} entityProto The protocol entity object to convert.
+   * @param {object} typeCastOptions Config for custom `integerValue` cast.
```

Comment:
Drop one line, and same concerns about where to bracket to denote optionality.

Reply:
Done